providers/fips/fipsprov.c

```diff
@@ -27,6 +27,12 @@
 #include "crypto/context.h"
 #include "internal/core.h"
 
+#ifdef OPENSSL_FIPS_PEDANTIC
+# define _PEDANTIC 1
+#else
+# define _PEDANTIC 0
```

Comment:
I think we normally don't use leading underscores in macro names to avoid Compiler/OS namespace pollution

Reply:
Leading underscore names are reserved by the standard I believe.

Reply:
and most OPENSSL macros are double-negative by default. And somehow I want this to not be confusing at all.

Reply:
There are going to be changes in this area once the indicator work is started based on the still evolving #23609

Reply:
@paulidale i'm trying to sidestep a few of those things.... by not compiling or providing any unapproved things at all. Because then, one doesn't need as many indicators.

Reply:
and also `-DPEDANTIC` is already a compiler thing